installer/scripts/omsadmin.sh

```diff
@@ -349,6 +349,21 @@ onboard_scom()
     make_dir $CONF_DIR/omsagent.d
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
+    save_config
```

Comment:
Thanks. I'll remove this call from onboard_scom (I shouldn't have assumed anyway), but for LAD, I wasn't able to run the service_control script without creating omsadmin.conf, so that's why I added. I still would like to rely on the service_control script to start/stop omsagent, so I thought I need omsadmin.conf anyway. Please correct me if I'm wrong.